src/watson/models.py

```diff
@@ -70,6 +70,9 @@ class SearchEntry(models.Model):
 
     meta_encoded = models.TextField()
 
+    def _deserialize_meta(self):
```

Comment:
ok, done, but slightly differently I've put `deserialize_meta` on `SearchEngine` not adapter to be the same as `serialize_meta`.